QuestionnaireResponse/QuestionnaireResponse-RoD-PheobeSmithamCDForm-Example.json

```diff
@@ -0,0 +1,157 @@
+{
+  "resourceType": "QuestionnaireResponse",
+  "id": "QuestionnaireResponse-RoD-PheobeSmithamCDForn-Example",
+  "questionnaire": "https://fhir.nhs.uk/Questionnaire/Questionnaire-RoD-ConsulteeDeclarationForm-Example",
+  "status": "completed",
+  "basedOn": [
+    {
+      "reference": "ServiceRequest/ServiceRequest-WGSTestOrderForm-TrioTestingProband-Example"
+    }
+  ],
+  "subject": {
+    "reference": "Patient/Patient-PheobeSmitham-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307539"
+    },
+    "display": "A Patient"
+  },
+  "authored": "2023-09-15",
+  "author": {
+    "type": "PractitionerRole",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/sds-role-profile-id",
+      "value": "999999999999"
+    }
+  },
+  "source": {
+    "type": "RelatedPerson",
+    "reference":"RelatedPerson/RelatedPerson-AliceSmithamProbandMother-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307246"
+    }
+  },
+  "item": [
+    {
+      "linkId": "patientDetails",
+      "text": "Patient Details",
+      "item": [
+        {
+          "linkId": "givenName",
+          "text": "First Name",
+          "answer": [
+            {
+              "valueString": "Phoebe"
+            }
+          ]
+        },
+        {
+          "linkId": "familyName",
+          "text": "Last Name",
+          "answer": [
+            {
+              "valueString": "Smitham"
+            }
+          ]
+        },
+        {
+          "linkId": "nhs_Number",
+          "text": "NHS number (or postcode if not not known)",
+          "answer": [
+            {
+              "valueString": "9449307539"
+            }
+          ]
+        },
+        {
+          "linkId": "birthDate",
+          "text": "Date of Birth",
+          "answer": [
+            {
+              "valueDate": "2013-09-27"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "confirmationOfDecision",
+      "text": "Confirmation of decision",
+      "item": [
+        {
+          "linkId": "confirmation",
```

Comment:
Missing Name of Consultee and Date (for Alice Smitham)

Reply:
Changed logic for Questionnaire-RoD-ConsulteeDeclarationForm-Example to allow recording of name and date when consent is taken remotely.